coffee-frontend/src/utils/loginUtils.js

```diff
@@ -46,28 +38,10 @@ const request = (options) => {
 };
 
 export function getCurrentUser() {
-    if(!localStorage.getItem(ACCESS_TOKEN)) {
-        return Promise.reject("No access token set.");
-    }
+    axios.defaults.headers.common['Authorization'] = 'Bearer ' + localStorage.getItem(ACCESS_TOKEN);
```

Comment:
This now sets the default header for all axios calls every time the user is fetched. Is this intended? Shouldn't it be enough to set the header once, when the user is authorized?

Reply:
In theory. The reason I set it here is that the user may already be authorized, and this is the first action that then is taken (if there already exists a token in localstorage)